lib/gym/package_command_generator.rb

```diff
@@ -32,6 +38,11 @@ def pipe
         [""]
       end
 
+      def postfix
+        # Remove the patched PackageApplication file after the export is finished
+        ["; rm -rf #{@patched_package_application}"]
```

Comment:
Why -r ? It's a file. This is risky if someone starts playing with the.

Why ; ? Maybe && so that we preserve the exit status of the command.

Finally, and to solve the 2 above issues, if you go for a temporary file, I would prefer using a random file path, created by Ruby mktemp, and automatically deleted by ruby (and thus getting rid off the shell rm command) to make sure one can run multiple builds from various processes concurrently. The above isn't CI friendly.